pkg/universe.dagger.io/alpha/terraform/import.cue

```diff
@@ -0,0 +1,18 @@
+package terraform
+
+// Run `terraform import`
+#Import: {
+	// The `address` of the specified Terraform resource to import
+	address: string
+
+	// The `id` of the specified Terraform `address` to import
+	id: string
+
+	#Run & {
+		// Terraform `import` command
+		cmd: "import"
+
+		// Adding the `address` and `id` as positional arguments 
+		cmdArgs: ["\(address)", "\(id)"]
```

Comment:
Why not directly `cmdArgs: [address, id]`? Something goes wrong if you do not interpolate?

Reply:
added your suggestions and the tests still pass.